### ISSUE TITLE
AoV 0.7.3: Visual effects messed up for caster class

### ISSUE THREAD

**Issue by Knito58** [Tamaized/AoV#35]
MC 1.12.2, Forge 14.23.0.2515, AoV 0.7.3, SSP
The caster's visual effects for searing light and other spells just do not work anymore. The searing light still kills but instead of a white line of particles there's a sparkling white cloud behind the caster which stays put in the air. All spells do work but the visual effects are messed up somehow.

**Comment by Tamaized** [comment]
Seems to be working fine?
![2017-11-14_09 40 24](https://user-images.githubusercontent.com/9671313/32795207-e3afb79c-c91f-11e7-9133-09350d81d680.png)

**Comment by Knito58** [comment]
Your image shows the spells as they should work. Something told me that starting over could be an effort that could help. You see, I tell my useers at work to start over if they have problems. So I restarted Minecraft from scratch and now it works for me, too. I did not change any setting and cannot reproduce the issue but have screenshots of the "clouds". The effects are back and working normally again.
